glitch/apps.py

```diff
@@ -15,31 +16,59 @@
     read_frame,
 )
 
+block_aspects = [
+    [1, 1],
+    [1, 4],
+    [4, 1]
+]
 
-def glitch_image(input_path: str, output_path: str) -> None:
+def glitch_image(input_path: str, output_path: str,
+            block_movement: float = 0.5, block_size: float = 0.5, block_count: int = 15,
+            noise_intensity: float = 0.5, noise_amount: float = 0.5,
+            channels_movement: float = 0.5) -> None:
     """ swaps some random blocks, random moves channels and adds salt and pepper noise to the image
     """
     image = imageio.imread(input_path)
 
-    image = move_random_blocks(
-        image, max_blocksize=(150, 400), num_blocks=3, per_channel=True
-    )
+    if block_count > 0 and block_size > 0 and block_movement > 0:
+        size  = int(min(image.shape[0], image.shape[1]) / 2 * block_size)
 
-    image = move_random_blocks(
-        image, max_blocksize=(200, 50), num_blocks=5, per_channel=True
-    )
+        blocks_moved = 0
 
-    image = move_channels_random(image, -10, 10)
+        while blocks_moved < block_count:
+            remaining_blocks = block_count - blocks_moved
+            
+            # Move blocks with given aspect ratio
+            num_blocks    = np.random.randint(1, remaining_blocks) if remaining_blocks > 1 else 1
 
-    image = salt_and_pepper(image, 0.75, 0.95)
+            blocks_moved += num_blocks
+
+            aspect = choice(block_aspects)
+            max_blocksize = [x * size for x in aspect]
+            image = move_random_blocks(
+                image,
+                max_blocksize=max_blocksize,
```

Comment:
so the aspect ratio is actually used as the `max_blocksize`. we could implement `move_random_block` (or a similar method) in a way that can directly take the desired aspect ratio